src/client/index.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { render } from 'react-dom';
+import App from './components/App';
+
+render(<App />, document.getElementById('container'));
```

Comment:
this won't work, you need to import `react-dom` and call `render` from it like this:
``` js
import ReactDOM from 'react-dom';
ReactDOM.render(
 <App />,
  document.getElementById('container')
);
```
and if you are going to use redux you will need to import `react-redux` here and provide the store like this: 
``` js
import { Provider } from 'react-redux';
import store from './store';
ReactDOM.render(
  <Provider store={store}>
    <App />
  </Provider>,
  document.getElementById('container')
);
```